devops/templates/pypi-variables.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
```

Comment:
This file is actually unused right now... I can't figure out how to use a template for variables (and the ADO documentation doesn't have a full example).

Reply:
If it's unused at the very least document that at the top of the file. Ideally let's not check in things that aren't used. As you may have noticed I'm not a huge fan of templating everything because it becomes unreadable.